src/main/groovy/de/dkfz/execution/jobs/cluster/pbs/PBSCommand.groovy

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2016 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.execution.jobs.cluster.pbs
+
+import de.dkfz.config.AppConfig
+import de.dkfz.eilslabs.tools.constants.StringConstants
+import de.dkfz.eilslabs.tools.logging.LoggerWrapper
+import de.dkfz.execution.jobs.Command
+import de.dkfz.execution.jobs.Job
+import de.dkfz.execution.jobs.ProcessingCommands
+
+import java.util.logging.Level
+
+/**
+ * This class is used to create and execute qsub commands
+ *
+ * @author michael
+ */
+@groovy.transform.CompileStatic
+public class PBSCommand extends Command {
+
+    private static final LoggerWrapper logger = LoggerWrapper.getLogger(PBSCommand.class.name);
+    public static final String NONE = "none"
+    public static final String NONE_ARR = "none[]"
+    public static final String AFTEROK = "afterok"
+    public static final String AFTEROK_ARR = "afterokarray"
+    public static final String QSUB = "qsub"
+    public static final String PARM_ACCOUNT = " -A "
+    public static final String PARM_JOBNAME = " -N "
+    public static final String PARM_JOINLOGS = " -j oe"
+    public static final String PARM_LOGPATH = " -o "
+    public static final String PARM_MAIL = " -M "
+    public static final String PARM_GROUPLIST = " -W group_list="
+    public static final String PARM_UMASK = " -W umask="
+    public static final String PARM_DEPENDS = " -W depend="
+    public static final String PARM_VARIABLES = " -v "
+    public static final String PARM_WRAPPED_SCRIPT = "WRAPPED_SCRIPT="
+
```

Comment:
NO! They are constants and constants are made this way! public static final String is an accepted standard.